src/06-loops.js

```diff
@@ -117,7 +159,16 @@ function targetCount(array, target) {}
  *  firstIndexFound([ "left", "right", "left" ], "up");
  *  //> -1
  */
-function firstIndexFound(array, target) {}
+function firstIndexFound(array, target) {
+  let foundIndex = -1;
+  for(let i = 0; i < array.length; i++){
+    if (array[i] === target){
+      foundIndex = i;
```

Comment:
We could shorten our code here to simply return `i` if we find the target.  This saves us from needing to write a break statement